test/test.ts

```diff
@@ -1,15 +1,13 @@
-import { expect } from 'chai';
-import 'mocha';
+import { expect } from 'chai'
+import 'mocha'
 
 /*tslint:disable*/
-const fixtureJson = require('./fixture.json');
-const outputJson = require('./out/api_data.json');
+const fixtureJson = require('./fixture.json')
+const outputJson = require('./out/api_data.json')
 /*tslint:enable*/
 
-describe('should compile', () => {
-
-  it('should compile api data correctly', () => {
-    expect(outputJson).to.deep.equal(fixtureJson);
-  });
-
-});
+describe('Apidoc TS Plugin', () => {
+  it('Should include TS info in compiled api docs', () => {
+    expect(outputJson).to.deep.equal(fixtureJson)
```

Comment:
These tests are really weird. It would be much natural to execute apidocs from within the test (in the worst case executing a new process, but ideally requiring apidocs from the process) and then compare the jsons. That way you could probably mock the files